installscripts/jazz-terraform-unix-noinstances/scripts/dynamodb.sh

```diff
@@ -38,6 +39,19 @@ case "$TABLE_FAMILY" in
 		aws dynamodb put-item --table-name $TABLE_NAME --item "{\"EVENT_NAME\":{\"S\":\"DELETE_PROJECT\"}}"
 		aws dynamodb put-item --table-name $TABLE_NAME --item "{\"EVENT_NAME\":{\"S\":\"BUILD\"}}"
 		aws dynamodb put-item --table-name $TABLE_NAME --item "{\"EVENT_NAME\":{\"S\":\"VALIDATE_PRE_BUILD_CONF\"}}"
+        aws dynamodb put-item --table-name $TABLE_NAME --item "{\"EVENT_NAME\":{\"S\":\"CREATE_BRANCH\"}}"
```

Comment:
Do you think this could have been done using batch-write-item?
https://docs.aws.amazon.com/cli/latest/reference/dynamodb/batch-write-item.html

Reply:
Possibly, but I didn't add this, I just merged it from `1.4.6`, if it can be optimized it should be optimized in a separate PR.

Reply:
ok